nautobot/extras/filters/mixins.py

```diff
@@ -48,6 +50,7 @@ def __init__(self, *args, **kwargs):
             CustomFieldTypeChoices.TYPE_INTEGER: CustomFieldNumberFilter,
             CustomFieldTypeChoices.TYPE_JSON: CustomFieldJSONFilter,
             CustomFieldTypeChoices.TYPE_MULTISELECT: CustomFieldMultiSelectFilter,
+            CustomFieldTypeChoices.TYPE_SELECT: CustomFieldSelectFilter,
```

Comment:
Should this be CustomFieldMultiSelectFilter? Not clear to me why we're using CustomFieldSelectFilter here but using CustomFieldMultiSelectFilter at line 100 below.

Reply:
Each class has a different `lookup expr`